cluster.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
```

Comment:
all files are in root directoy. it'll be cleaner to keep in `src` and only `cluster.js` remains outside as an entry point

Reply:
within `src` also, let's have a sensible directory structure.

Reply:
added lib instead of src